megamek/src/megamek/server/Server.java

```diff
@@ -2452,9 +2452,9 @@ else if (((infMoved && infMoveMulti)
 
         // move along
         if (outOfOrder) {
-            send(createTurnIndexPacket(entityUsed.getOwnerId()));
+            send(createTurnIndexPacket(playerId));
```

Comment:
playerId is earlier set to entityUsed.getOwnerId(), except it checks for entityUsed == null and sets playerId to -1 if it is.